public/app/features/explore/LogsVolumePanel.tsx

```diff
@@ -46,22 +56,51 @@ function ErrorAlert(props: { error: DataQueryError }) {
   );
 }
 
+type MergedInfo = {
+  logsVolumeData?: DataQueryResponse;
+  logLinesBased: boolean;
```

Comment:
Also here, I am wondering if we should rather do `fullRangeData`. To again emphasise that it has higher priority?

Reply:
sorry, i don't think that could work. this name `logsVolumeData` is the "final" data that is either full-range or logs-based. do you think we can keep the name?

Reply:
ok, i misunderstood what you meant, will update the code 👍

Reply:
done.